### PR TITLE
Added tooltip to Awards and Milestones to inform about cost using <abbr>

### PR DESCRIPTION
**Pull request by peonmodel**
added <abbr> tag to milestone and awards to remind players about the cost

added support for svg images

added support for changing HOST ip address as per instructions in README

### CODE REVIEW

File: src/components/Award.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ export const Award = Vue.component("award", {
     <div class="awards_cont" v-trim-whitespace>
         <div class="awards">
             <div class="ma-title">
-                <a  class="ma-clickable awards-padding" href="#" v-on:click.prevent="toggleMe()" v-i18n>Awards</a>
+                <a class="ma-clickable awards-padding" href="#" v-on:click.prevent="toggleMe()" v-i18n>
+                    <abbr title="costs 8 | 14 | 20 M€" tabindex="0">Awards<abbr>
```
Comment (chosta) [inline src/components/Award.ts:36]: Does the translation for different languages work for the title?

**Review by chosta — APPROVED** [review]
I personally avoid hover because of mobile but it's an improvement to an extent.

File: src/components/Milestone.ts [hunk 2]
```diff
@@ -33,7 +33,9 @@ export const Milestone = Vue.component("milestone", {
     <div class="milestones_cont" v-trim-whitespace>
         <div class="milestones">
             <div class="ma-title">
-                <a class="ma-clickable" href="#" v-on:click.prevent="toggleMe()" v-i18n>Milestones</a>
+                <a class="ma-clickable" href="#" v-on:click.prevent="toggleMe()" v-i18n>
+                    <abbr title="costs 8 M€" tabindex="0">Milestones<abbr>
```
Comment (bafolts) [inline src/components/Milestone.ts:37]: This should have a close for `</abbr>`

**Comment by peonmodel** [comment]
i fixed the typo in the end tag </abbr>

heres a screenshot for how the tooltip looked like on mobile

Screenshot_20201018-105845_Brave

File: src/components/Award.ts (hunk 1 above)
Comment (alrusdi) [reply src/components/Award.ts]: Yeah! Please make the title translatable

File: server.ts [hunk 3]
```diff
@@ -1015,6 +1015,16 @@ function serveAsset(req: http.IncomingMessage, res: http.ServerResponse): void {
         }
         res.setHeader("Content-Type", "image/png");
         file = reqFile;
+    } else if (req.url.endsWith(".svg")) {
```
Comment (alrusdi) [inline server.ts:1018]: Why do we need it? Is it from some other PR?

Comment (peonmodel) [reply server.ts:1018]: it is not required for the tooltip

Comment (alrusdi) [reply server.ts:1018]: then it should be excluded from this PR

**Comment by kberg** [comment]
This can't be accepted until the conflicts are resolved. If you think that's something you can get to soon, that's great. If you need some help, definitely ask!

**Comment by peonmodel** [comment]
the tooltip issue has been resolved by another update, shall close this pull request

**Comment by kberg** [comment]
Ah, thanks!

**Pull request closed without merging**